### ISSUE TITLE
Trouble installing on OSX

### ISSUE THREAD

**Issue by jcmorrow** [ellipticoin/ec-wallet#1]
Hitting a snag while following https://ellipticoin.readthedocs.io/en/latest/developers/#deploying-a-smart-contract.

```
$ npm install -g ec-wallet
npm WARN deprecated bignumber.js@6.0.0: Custom ALPHABET bug fixed in v7.0.2
/usr/local/bin/ec-wallet -> /usr/local/lib/node_modules/ec-wallet/ts-built/cli.js

> ed25519@0.0.4 install /usr/local/lib/node_modules/ec-wallet/node_modules/ed25519
> node-gyp rebuild

gyp ERR! configure error 
gyp ERR! stack Error: Command failed: /Users/josh/.pyenv/shims/python -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                                ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack 
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:297:12)
gyp ERR! stack     at ChildProcess.emit (events.js:203:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:1021:16)
gyp ERR! stack     at Socket.<anonymous> (internal/child_process.js:430:11)
gyp ERR! stack     at Socket.emit (events.js:203:13)
gyp ERR! stack     at Pipe.<anonymous> (net.js:588:12)
gyp ERR! System Darwin 18.2.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/ec-wallet/node_modules/ed25519
gyp ERR! node -v v12.6.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ed25519@0.0.4 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ed25519@0.0.4 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/josh/.npm/_logs/2019-07-17T22_11_37_519Z-debug.log
```

This *seems* to be a python2 vs. python3 problem, but I can't entirely be sure. I've tried pointing `node-gyp rebuild` to my python 2.7 installation, but with no luck 😞 

Thoughts?

**Comment by masonforest** [comment]
Oops I'm not using that dependency anymore. This should be fixed in `0.4.10`

**Comment by jcmorrow** [comment]
Fixed for me!
